packages/ui/test/enzyme.card.test.js

```diff
@@ -38,3 +41,21 @@ test('Should display given content', t => {
   const wrapper = shallow(<Card content='TEXT' />);
   t.is(wrapper.find('p').text(), 'TEXT');
 });
+
+test('Test track number hover shows play icon', t => {
```

Comment:
This is a file for card tests. Could you please move this test to the file for TrackRow component?

Note that this package (ui) has its own TrackRow, which will be merged with the one from the app package in the future. So you'll want to move your tests here: https://github.com/nukeop/nuclear/tree/master/packages/app/test

You could name it `enzyme.trackrow.test.js` or something similar.